testutil/validatormock/attest.go

```diff
@@ -80,6 +80,7 @@ func (a *SlotAttester) Slot() eth2p0.Slot {
 // - Fetches attester attDuties for the slot (this could be cached at start of epoch).
 // - Prepares aggregation attDuties for slot attesters.
 // It panics if called more than once.
+// TODO(xenowits): Fix off-chance race condition bug when Prepare panics if called for the same slot multiple times.
```

Comment:
this should never happen... it wasn't designed to be called twice

Reply:
So the comment should be: TODO: figure out why this is sometimes called twice (which causes panics)

Reply:
created a ticket here: https://github.com/ObolNetwork/charon/issues/1389 to solve this. I'll fix this in the next PR